packages/website/src/components/Playground.tsx

```diff
@@ -70,6 +55,25 @@ function Playground(): JSX.Element {
     [],
   );
 
+  const activeVisualEditor = useMemo(() => {
+    if (!isLoading) {
+      return visualEslintRc && activeTab === 'eslintrc'
+        ? 'eslintrc'
+        : visualTSConfig && activeTab === 'tsconfig'
+        ? 'tsconfig'
+        : undefined;
+    }
+    return undefined;
+  }, [activeTab, isLoading, visualEslintRc, visualTSConfig]);
```

Comment:
[Performance] I don't think this is perf-intensive enough to justify a `useMemo`?

Reply:
agree, initially there was way more code in there :)